torch/onnx/_onnx_supported_ops.py

```diff
@@ -75,18 +72,27 @@ def _symbolic_argument_count(func):
     return params
 
 
-def _all_symbolics_schemas():
-    symbolics_schemas: Dict[str, _TorchSchema] = {}
+def _all_symbolics_schemas() -> Dict[str, _TorchSchema]:
```

Comment:
I guess this change should be made in the next PR? #84496

Reply:
We need to change `_all_symbolics_schemas` because of the new registry so might as well make the function right the first go